app/bundles/EmailBundle/Swiftmailer/Momentum/Service/SwiftMessageService.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Mautic\EmailBundle\Swiftmailer\Momentum\Service;
+
+use Mautic\EmailBundle\Helper\PlainTextMassageHelper;
+use Mautic\EmailBundle\Swiftmailer\Message\MauticMessage;
+use Mautic\EmailBundle\Swiftmailer\Momentum\DTO\TransmissionDTO;
+use Mautic\EmailBundle\Swiftmailer\Momentum\Metadata\MetadataProcessor;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Class SwiftMessageService.
+ */
+final class SwiftMessageService implements SwiftMessageServiceInterface
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * MomentumSwiftMessageService constructor.
+     *
+     * @param TranslatorInterface $translator
+     */
+    public function __construct(
+        TranslatorInterface $translator
+    ) {
+        $this->translator = $translator;
+    }
+
+    /**
+     * @param \Swift_Mime_Message $message
+     *
+     * @return TransmissionDTO
+     */
+    public function transformToTransmission(\Swift_Mime_Message $message)
+    {
+        $messageFrom      = $message->getFrom();
+        $messageFromEmail = current(array_keys($messageFrom));
+        $from             = new TransmissionDTO\ContentDTO\FromDTO($messageFromEmail);
+        if (!empty($messageFrom[$messageFromEmail])) {
+            $from->setName($messageFrom[$messageFromEmail]);
+        }
+        $content = new TransmissionDTO\ContentDTO($message->getSubject(), $from);
+
+        $metadataProcessor = new MetadataProcessor($message);
```

Comment:
There is a parameter mismatch - The class expect `Swift_Message` and you provided `Swift_Mime_Message`.